src/js/action-types.js

```diff
@@ -16,11 +16,35 @@ export const SET_UPLOAD_DATA_TYPES = 'SET_UPLOAD_DATA_TYPES'
 export const SET_UPLOAD_DATA_TYPE_BY_HEADER_NAME = 'SET_UPLOAD_DATA_TYPE_BY_HEADER_NAME'
 export const SET_VISUALIZATION = 'SET_VISUALIZATION'
 
+export const DASHBOARD_CREATE_VISIBILITY = 'DASHBOARD_CREATE_VISIBILITY'
+export const DASHBOARD_CREATE_SUBTITLE_FORM = 'DASHBOARD_CREATE_SUBTITLE_FORM'
+export const DASHBOARD_CREATE_TITLE_FORM = 'DASHBOARD_CREATE_TITLE_FORM'
+export const DASHBOARD_CREATE_SAVE = 'DASHBOARD_CREATE_SAVE'
+export const DASHBOARD_DELETE_VISIBILITY = 'DASHBOARD_DELETE_VISIBILITY'
+export const DASHBOARD_EDIT_POPULATE_FORM = 'DASHBOARD_EDIT_POPULATE_FORM'
+export const DASHBOARD_EDIT_SUBTITLE_FORM = 'DASHBOARD_EDIT_SUBTITLE_FORM'
+export const DASHBOARD_EDIT_TITLE_FORM = 'DASHBOARD_EDIT_TITLE_FORM'
+export const DASHBOARD_EDIT_VISIBILITY = 'DASHBOARD_EDIT_VISIBILITY'
+export const DASHBOARD_EDIT_SAVE = 'DASHBOARD_EDIT_SAVE'
+export const DASHBOARD_SELECT = 'DASHBOARD_SELECT'
+
 // ASYNC
 
 export const FETCH_ANALYTICS_FAILURE = 'FETCH_ANALYTICS_FAILURE'
 export const FETCH_ANALYTICS_REQUEST = 'FETCH_ANALYTICS_REQUEST'
 export const FETCH_ANALYTICS_SUCCESS = 'FETCH_ANALYTICS_SUCCESS'
+export const FETCH_DASHBOARD_CREATE_FAILURE = 'FETCH_DASHBOARD_CREATE_FAILURE'
```

Comment:
This seems like a strange action name?

If it is actually "fetching" a dashboard, than you shouldn't need `FETCH_DASHBOARD_CREATE` and `FETCH_DASHBOARD_EDIT` since they would just do the same thing.

However, if this is actually the post that creates the dashboard, than we may want to think about our wording more.

`CREATE_DASHBOARD_(FAILURE|REQUEST|SUCCESS)`?